src/main/scala/codacy/plugins/test/serializers/Serializers.scala

```diff
@@ -0,0 +1,30 @@
+package codacy.plugins.test.serializers
+
+import com.codacy.plugins.api.results.Result
+import com.codacy.plugins.api.results.Result._
+import com.codacy.plugins.api.Source
+import com.codacy.plugins.api.results.Pattern
+import io.circe._
+
+object Serializers {
+  implicit val issueDecoder: Decoder[Issue] = new Decoder[Issue] {
+    final def apply(c: HCursor): Decoder.Result[Issue] =
+      for {
+        filename <- c.downField("filename").as[String]
+        message <- c.downField("message").as[String]
+        patternId <- c.downField("patternId").as[String]
+        line <- c.downField("line").as[Int]
+      } yield Result.Issue(file = Source.File(filename), message = Result.Message(message), patternId = Pattern.Id(patternId), line = Source.Line(line))
+  }
+
+  implicit val extendedIssueDecoder: Decoder[ExtendedIssue] = ???
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of ??? operator](https://app.codacy.com/gh/codacy/codacy-plugins-test/pullRequest?prid=5569494)